wrappers/rust/icicle-core/src/msm/tests.rs

```diff
@@ -1,5 +1,6 @@
-use super::{get_default_msm_config, msm, MSM};
+use super::{msm, MSM};
 use crate::curve::{Affine, Curve, Projective};
+use crate::msm::MSMConfig;
```

Comment:
Maybe unify imports here to either
```rust
use super::{msm, MSM, MSMConfig};
```
or
```rust
use crate::msm::{msm, MSM, MSMConfig};
```

Reply:
fixed